indigo_app/templates/indigo_api/_task_detail.html

```diff
@@ -3,6 +3,23 @@
 <div class="container mt-3 mb-5">
     <div class="row">
       <div class="mb-3 col-9">
+        {% if task.state == 'blocked' %}
+          <div class="alert alert-danger">
+            This task is blocked.
+            {% with task.blocked_by.all as blocked_by %}
+              {% if blocked_by %}
+                These tasks must be completed first:
+                <ul class="mb-2">
```

Comment:
```suggestion
                <ul class="mt-2">
```

The alert has inner padding, so rather give some space between the intro sentence and the list.